index.html

```diff
@@ -77,9 +78,6 @@
                         }`}
                         onClick={() => {
                             setActive(d.id)
-                            setCurrentContent(
-                                documents.find((d2) => d2.id == d.id).content,
-                            )
```

Comment:
Because `currentContent` is derived state from `active`, `setActive` is sufficient here.